### PR TITLE
Filter more errorProperties

### PR DESCRIPTION
**Pull request by formulahendry**

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -194,9 +194,13 @@ export class Constants {
         [TemplateLanguage.Ruby]: "ruby",
     };
 
+    // 'error', 'errorMessage' and 'stack' are used in vscode-azureextensionui npm
```
Comment (blackchoey) [inline src/constants.ts:197]: Can we control the telemetry in vscode-azureextensionui package? Maybe we need to ask the maintainer for an update?

File: src/simulator.ts [hunk 2]
```diff
@@ -99,7 +99,7 @@ export class Simulator {
     if (eventName === Constants.SimulatorLaunchEvent) {
       TelemetryClient.sendEvent(eventName, {
         Result: result ? "Success" : "Fail",
-        Error: result ? undefined : properties.error,
+        [Constants.errorProperties.Message]: result ? undefined : properties.error,
```
Comment (blackchoey) [inline src/simulator.ts:102]: Please also update property name in other events if you want to rename this property. For example, line 117 in this file.

Comment (formulahendry) [reply src/simulator.ts:102]: Good catch!

File: src/constants.ts (hunk 1 above)
Comment (formulahendry) [reply src/constants.ts:197]: We could not control the telemetry property in vscode-azureextensionui package. I don't think we need to ask them to change telemetry property name.

**Pull request merged**